File: https-quera.org-problemset-3414/Solution.py
def app():
    nums = tuple(map(int, input().split()))
    x1, y1, x2, y2 = nums[0], nums[1], nums[2], nums[3]

    if x1 == x2:
        print("Vertical")
    elif y1 == y2:
        print("Horizontal")
    else:
        print("Try again")


if __name__ == "__main__":
    app()

File: https-quera.org-problemset-3414/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }


    public static void app() {
        Scanner scanner = new Scanner(System.in);
        
        String[] input = scanner.nextLine().split(" ");
        
        int x1 = Integer.parseInt(input[0]);
        int y1 = Integer.parseInt(input[1]);
        int x2 = Integer.parseInt(input[2]);
        int y2 = Integer.parseInt(input[3]);

        if (x1 == x2) {
            System.out.println("Vertical");
        } else if (y1 == y2) {
            System.out.println("Horizontal");
        } else {
            System.out.println("Try again");
        }
    }

}
